python/tvm/autotvm/tuner/xgboost_tuner.py

```diff
@@ -40,16 +40,21 @@ class XGBTuner(ModelBasedTuner):
         If is not None, the tuner will first select
         top-(plan_size * diversity_filter_ratio) candidates according to the cost model
         and then pick batch_size of them according to the diversity metric.
+    verbose: int
```

Comment:
consider directly rely on logging level for verbosity

Reply:
This is int not bool, so we leave this.

Reply:
verbose usually do not have the same meaning, so this argument is confusing. A better name is log_interval.